docs/resources/Invite.md

```diff
@@ -83,7 +83,7 @@ Extra information about an invite, will extend the [invite](#DOCS_RESOURCES_INVI
 
 ## Get Invite % GET /invites/{invite.code#DOCS_RESOURCES_INVITE/invite-object}
 
-Returns an [invite](#DOCS_RESOURCES_INVITE/invite-object) object for the given code.
+Returns an [invite](#DOCS_RESOURCES_INVITE/invite-object) object for the given code. Doesn't require authentication.
```

Comment:
This should say `Authorization`, since `Authentication` is the check that you have permission to view the resource once you have authorized with a valid token (which is internal server logic we don't see anyway)